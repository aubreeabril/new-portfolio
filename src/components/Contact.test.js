import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Contact from "./Contact";

describe("Contact", () => {
  it("renders the Contact section", () => {
    const wrapper = shallow(<Contact />);
    expect(wrapper.find(".contact-text").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    const section = renderer.create(<Contact />).toJSON();
    expect(section).toMatchSnapshot();
  });
});
