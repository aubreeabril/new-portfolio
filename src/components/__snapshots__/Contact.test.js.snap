// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact matches its snapshot 1`] = `
<div
  className="contact-text"
>
  <a
    href="https://twitter.com/aubreeabril"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      className="fab fa-twitter"
    />
  </a>
  <a
    href="https://github.com/aubreeabril"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      className="fab fa-github"
    />
  </a>
  <a
    href="mailto:aubree.abril@gmail.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      className="fas fa-envelope"
    />
  </a>
</div>
`;
